File: SegurityAngular-master/src/app/model/TransferToken.ts
export class TransferToken{
    transferWalletId:number;
    receptorWalletId:number;
    tokenName:string;
    amount:number;
    constructor(transferWalletId:number,
        receptorWalletId:number,
        tokenName:string,
        amount:number
    ){
        this.amount=amount;
        this.receptorWalletId=receptorWalletId;
        this.tokenName=tokenName;
        this.transferWalletId=transferWalletId;
    }
}

File: G4/src/main/java/com/bootcamp/G4/model/TransferToken.java
package com.bootcamp.G4.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TransferToken {
    private Long transferWalletId;

    private Long receptorWalletId;

    private String tokenName;

    private double amount;
}
